app/controllers/chat_channels_controller.rb

```diff
@@ -64,20 +64,19 @@ def unfollow
   def create
     params.require([:type, :id, :name])
     guardian.ensure_can_create_chat_channel!
-    raise Discourse::InvalidParameters unless ["topic", "category"].include?(params[:type].downcase)
+    raise Discourse::InvalidParameters unless ["category"].include?(params[:type].downcase)
```

Comment:
Could just be an == now that there's only 1 value

Reply:
Yes sure